contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -245,15 +685,58 @@ NXmpes(NXobject):
           varied axis may be for example energy, momentum, spatial coordinate, pump-probe
           delay, spin index, temperature, etc. The axes traces should be linked to the
           actual encoder position in NXinstrument or calibrated axes in NXprocess.
+      energy(NX_NUMBER):
+        unit: NX_ENERGY
+        doc: |
+          Calibrated energy axis.
+          
+          This could be a link to either
+          /entry/process/energy_calibration/calibrated_axis or
+          /entry/process/energy_correction/calibrated_axis.
+        \@type:
+          type: NX_CHAR
+          doc: |
+            The energy can be either stored as kinetic or as binding energy.
+          enumeration: 
+            kinetic: 
+              doc:
+              - |
+                Calibrated kinetic energy axis.
+              - |
+                xref:
+                  spec: ISO 18115-1:2023
+                  term: 3.35
+                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.35
+            binding: 
+              doc:
+              - |
+                Calibrated binding energy axis.
+              - |
+                xref:
+                  spec: ISO 18115-1:2023
+                  term: 12.16
+                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
+      \@energy_indices:
+        exists: recommended
+      \@energy_depends:
+        type: NX_CHAR
+        exists: recommended
+        doc: |
+          The energy can be dispersed according to different strategies. ``energy_depends`` points to
+          the path of a field defining the axis on which the energy axis depends.
+          
+          For example:
+            @energy_depends: '/entry/instrument/detector/data/sensor_y' for a 2D detector
+            @energy_depends: '/entry/instrument/energydispersion/center_energy' for a swept scan
```

Comment:
I would say this should rather point to the field determining the final calibrated axis, e.g. /entry/process/energy_calibration. This also contains a link then to the raw data, but allows to understand what the energy axis represents.

Reply:
I changed the doc to:
```yaml
doc: |
  The energy can be dispersed according to different strategies. ``energy_depends`` points to
  the path of a field defining the calibrated axis on which the energy axis depends.
  
  For example:
    @energy_depends: 'entry/process/energy_calibration'
```

Reply:
sounds good.